### PR TITLE
Add public slack logs to readme

### PR DESCRIPTION
**Pull request by MaxWofford**

### CODE REVIEW

**Comment by MaxWofford** [comment]
@zachlatta Don't merge this just

File: README.md [hunk 1]
```diff
@@ -26,7 +26,10 @@ check out and contribute to this repository.
 Important links:
 
 - [Workshops](workshops/README.md) :: Workshops for use in clubs
-- https://slack.hackclub.io :: Join our [Slack](https://slack.com/) community
+- https://slack.hackclub.io :: Join
+  [our Slack](https://starthackclub.slack.com/) community
```
Comment (zachlatta) [inline README.md]: Err, was there a reason why you linked `our Slack`? The `https://slack.hackclub.io` link is already there.

File: README.md [hunk 2]
```diff
@@ -26,7 +26,10 @@ check out and contribute to this repository.
...
+- Our public [chat logs](http://starthackclub.slackarchive.io/) if you want to
+  see what we're talking about
```
Comment (zachlatta) [inline README.md]: I might make this a complete sentence.

Also, what are your thoughts on just leaving this in the topic of #introductions or #announcements? It feels a little out of place in at the top of our README (which could be thought of as our landing page).

File: README.md (hunk 1 above)
Comment (MaxWofford) [inline README.md]: @zachlatta Rather than linking to _slack_ the company, I thought it was more relevant to link to our slackin.

**Comment by MaxWofford** [comment]
This was announced in the #announcements channel.

File: README.md (hunk 1 above)
Comment (zachlatta) [inline README.md]: Ah, but the Slackin is already linked on the left, right? (the `https://slack.hackclub.io :: ` bit).

I think it's worthwhile to have a link to Slack itself in there for people who don't know what Slack is.

**Comment by MaxWofford** [comment]
Closing this because https://github.com/hackclub/hackclub/pull/525/files#r46340964

**Pull request closed without merging**